mirage/apt/apt_inputs.py

```diff
@@ -52,6 +52,7 @@
 
 from . import read_apt_xml
 from ..utils import siaf_interface
+from ..utils import constants
```

Comment:
I think you did not commit the constants.py file.

Reply:
Sorry about that - just added it.